content/en/cosign/sign.md

```diff
@@ -161,19 +161,112 @@ The upload is skipped by using the `--upload=false` flag (default true). To capt
 $ cosign sign --key key.pem --upload=false --output-signature demo.sig --output-certificate demo.crt user/demo
 ```
 
-## Generate the signature payload (to sign with another tool)
+## Generate the Signature Payload With `cosign` (to sign with another tool)
```

Comment:
I would write out "Cosign" vs. using the command here , and would make the "with" lower case

```suggestion
## Generate the Signature Payload with Cosign (to sign with another tool)
```